client/webserver/site/src/html/orders.tmpl

```diff
@@ -0,0 +1,81 @@
+{{define "orders"}}
+{{template "top" .}}
+<div id="main" data-handler="orders" class="main overflow-y-auto">
+
+  <div class="w-100 d-flex justify-content-start align-items-stretch">
+    <div class="d-flex flex-column align-items-stretch justify-content-start orders-left-side">
+      <div class="demi fs22 text-center py-1 px-4">Order History</div>
+      <div class="filter-display bg1">Exchanges</div>
+      <div id="hostFilter" class="filter-opts">
+        {{range .Hosts}}
+        <div class="pl-4 pr-2">
+          <input class="form-check-input" type="checkbox" value="{{.}}">
+          <label class="form-check-label fs14">{{.}}</label>
+        </div>
+        {{end}}
+        <div class="apply-bttn d-hide"><button class="bg2 demi">apply</button></div>
+      </div>
+      <div class="filter-display bg1">Assets</div>
+      <div id="assetFilter" class="filter-opts">
+        {{range $assetID, $asset := .Assets}}
+        <div class="pl-4 pr-2">
+          <input class="form-check-input" type="checkbox" value="{{$assetID}}">
+          <label class="form-check-label fs14">{{$asset.Symbol}}</label>
+        </div>
+        {{end}}
+        <div class="apply-bttn d-hide"><button class="bg2 demi">apply</button></div>
+      </div>
+      <div class="filter-display bg1">Status</div>
+      <div id="statusFilter" class="filter-opts">
+        {{range $status, $statusString := .Statuses}}
+        <div class="pl-4 pr-2">
+          <input class="form-check-input" type="checkbox" value="{{$status}}">
+          <label class="form-check-label fs14">{{$statusString}}</label>
+        </div>
+        {{end}}
+        <div class="apply-bttn d-hide"><button class="bg2 demi">apply</button></div>
+      </div>
+    </div>
+    <div class="flex-grow-1">
+      <table class="orderstable" id="ordersTable">
+        <thead id="tableHead" class="bg1 fs16">
+          <tr>
+            <th>Trade</th>
+            <th>Market</th>
+            <th>Type</th>
+            <th>Status</th>
+            <th class="text-right">Rate</th>
+            <th class="text-right">Filled</th>
+            <th class="text-right">Settled</th>
+            <th></th> {{/* Link */}}
```

Comment:
I think we need some kind of date for each order.

Reply:
A trade token might be helpful too.  e.g. I'm looking at details for a single order in the /order/{oid} page, then go back to the orders page and I don't know which one it was I was just looking at. I was doing this when looking back and forth between maker/taker sides of a match, but still.

Reply:
Sort order appears to be random, even changes.